index.js

```diff
@@ -155,27 +155,28 @@ export function execa(rawFile, rawArgs, rawOptions) {
 	const controller = new AbortController();
 	setMaxListeners(Number.POSITIVE_INFINITY, controller.signal);
 
+	const originalStreams = [...spawned.stdio];
```

Comment:
Transforms directly modify `spawned.stdin|stdout|stderr`. We need to keep it track of the original values.